modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -235,7 +235,56 @@ public void testGetProcessInstancesPageWithSortString() throws Exception {
 			(entityField, instance1, instance2) -> {
 				String entityFieldName = entityField.getName();
 
-				if (StringUtil.equals("userName", entityFieldName)) {
+				if (StringUtil.equals("assigneeName", entityFieldName)) {
+					instance1.setAssignees(
+						() -> {
+							User user = UserTestUtil.addUser(
+								RandomTestUtil.randomString(
+									NumericStringRandomizerBumper.INSTANCE,
+									UniqueStringRandomizerBumper.INSTANCE),
+								LocaleUtil.getDefault(),
+								"aaa".concat(
+									StringUtil.toLowerCase(
+										RandomTestUtil.randomString())),
+								RandomTestUtil.randomString(),
+								new long[] {TestPropsValues.getGroupId()});
+
```

Comment:
Hey @MariMalta and @seltong,

please extract the logic to create a user to a private method and apply to usages. The signature must be `private User _addUser(String firstName) throws Exception;`. 

Thx